recipes/ff/meta.yaml

```diff
@@ -0,0 +1,80 @@
+{% set name = "ff" %}
+{% set version = "2.0.0" %}
+
+package:
+  name: {{ name }}-static
+  version: {{ version }}
+
+source:
+    url: https://github.com/hep-packaging-coordination/ff/archive/v{{ version }}.tar.gz
+    sha256: 2329b3ca147825b660e07dac45ed85d397a1bea2f6b886518eb4d09900da5503
+    patches:
+      # Can be removed after the patched QCDLoop version is merged in v2.0.1.
+      - update-makefile.patch
+      # c.f. https://github.com/hep-packaging-coordination/ff/pull/8
+      - allow-longer-paths-for-ffopen.patch
+
+build:
+  skip: true  # [win]
+  number: 0
+  script:
+    # FIXME: Patch out custom paths
+    # Note: Need the space at the end of '$PREFIX/share/ff/ ' so that
+    #
+    # fullname = path(1:index(path,' ')-1)//names
+    #
+    # in src/ff/ffinit.f can have a space to split on.
+    # This is a bug and should get fixed by using better logic for fullname.
+    - sed -i "s|/user/gj/lib/|$PREFIX/share/ff/ |g" src/ff/ffinit.f
+    - sed -i "s|/usr/local/ff/|$PREFIX/share/ff/ |g" src/ff/ffinit.f
+
+    - cd src/ff
+    - make install FC=$FC FFLAGS="$FFLAGS -std=legacy -ffixed-line-length-none" DEST=$PREFIX/lib
+    - mkdir -p $PREFIX/include/ff
+    - mv *.h $PREFIX/include/ff
+    - mkdir -p $PREFIX/share/ff
+    - mv *.dat $PREFIX/share/ff/
+
+requirements:
+  build:
+    - {{ stdlib('c') }}
+    - {{ compiler('fortran') }}
+    - make
+    - libtool
+    - sed
+
+test:
+  source_files:
+    - src/ff/npointes.f
+  requires:
+    - {{ compiler('fortran') }}
+  commands:
+    - test -f $PREFIX/lib/libff.a
+    - test -f $PREFIX/include/ff/aa.h
+    - test -f $PREFIX/include/ff/ff.h
+    - test -f $PREFIX/include/ff/ffs.h
+    - test -f $PREFIX/share/ff/fferr.dat
+    - test -f $PREFIX/share/ff/ffperm5.dat
+    - test -f $PREFIX/share/ff/ffwarn.dat
+
+    - $FC ./src/ff/npointes.f -o npointes $FFLAGS -I$PREFIX/include/ff $LDFLAGS -lff
+    - ./npointes
```

Comment:
Things are now working as expected for Linux, but x86 macOS is now giving

```
 open: error: could not open /Users/runner/bld/ffwarn.dat                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    
         adjust path in ffopen (ffinit.f)
ffwarn: warning cannot open ffwarn.dat with warning texts
```

indicating that [`fullname`](https://github.com/hep-packaging-coordination/ff/blob/441ea0fffb6d4af12b15545f72e159c84256fa53/src/ff/ffinit.f#L779) is not being set as `$PREFIX/share/ff/ffwarn.dat` as expected above.

Reply:
The build also works as expected for 

```
pixi run build-osx osx_arm64
```

It just seems to be `x86` macOS that is having an issue now with the example.